test/jsonapi/plugs/deserializer_test.exs

```diff
@@ -0,0 +1,105 @@
+defmodule JSONAPI.DeserializerTest do
+  use ExUnit.Case
+  use Plug.Test
+
+  defmodule ExamplePlug do
+    use Plug.Builder
+    plug(Plug.Parsers, parsers: [:json], json_decoder: Jason)
```

Comment:
Style: I don't think we should use parens around plug calls because they exclude those in their own documentation (and formatter config)